students/didacbigorda/index.html

```diff
@@ -10,31 +10,78 @@
   <meta property="og:description" content="This is Didac Bigorda's personal page">
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  <link rel="stylesheet" href="styles/....">
+  <link rel="stylesheet" href="style.css">
 </head>
 
 <body itemscope itemtype="https://schema.org/Person">
   <header>
-    <h1>Dídac Bigordà Arruego</h1>
-    <h2>Frontend Developer</h2>
+    <nav class="top-nav">
+      <div class="nav-div">
+        <h1>Dídac Bigordà</h1>
+        <h2>Frontend Developer</h2>
+        <a href="https://tawebbcn.github.io/cohort0191/#">HOME</a>
+      </div>
+    </nav>
   </header>
   <main>
-    <section>
+    <section class="first-section">
+      <h1>
+        I'm Dídac, a web developer from Barcelona
+      </h1>
```

Comment:
You don't have to break everything. If you want you can leave in one line the h1 and h2 and li and a.